backend/src/main/kotlin/rocks/didit/sefilm/domain/dto/BuyDTO.kt

```diff
@@ -1,9 +1,11 @@
 package rocks.didit.sefilm.domain.dto
 
 import rocks.didit.sefilm.database.entities.ParticipantInfo
+import rocks.didit.sefilm.database.entities.Showing
 import rocks.didit.sefilm.domain.Bioklubbnummer
 
 data class BuyDTO(val bioklubbnummer: Collection<Bioklubbnummer>,
                   val sfBuyLink: String?,
-                  val participantInfo: Collection<ParticipantInfo>) {
+                  val participantInfo: Collection<ParticipantInfo>,
+                  val showing: Showing) {
```

Comment:
Probably, as it is now participantInfo seems to be empty and the frontend resolves to using an existing showing instead. Not sure if that is intentional